docs/connect.html

```diff
@@ -309,7 +311,7 @@ <h5><a id="connect_connectorexample" href="#connect_connectorexample">Connector
     }
     </pre>
 
-    <p>We will define the <code>FileStreamSourceTask</code> class below. Next, we add some standard lifecycle methods, <code>start()</code> and <code>stop()</code></p>:
+    <p>We will define the <code>FileStreamSourceTask</code> class below. Next, we add some standard lifecycle methods, <code>start()</code> and <code>stop()</code>:</p>
```

Comment:
fix wrong html format for the colon `:`.
before:
![image](https://user-images.githubusercontent.com/43372967/82002522-8d66c100-9690-11ea-8df3-8871569d188c.png)